freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/dns/doc/DnsModelDescription.java

```diff
@@ -0,0 +1,15 @@
+package com.sequenceiq.freeipa.api.v1.dns.doc;
+
+public final class DnsModelDescription {
+    public static final String DNS_RECORD = "DNS record name without the domain. eg. 'ipaserver' from 'ipaserver.clouder.site'";
+
+    public static final String DNS_ZONE = "It's the domain. Like if your FQDN is ipaserver.clouder.site, it's 'cloudera.site'. "
+            + "'168.192.in-addr.arpa' for a reverse record like '5.1.168.192.in-addr.arpa'";
+
+    public static final String CNAME_TARGET_FQDN = "The fully qualified domain name of the host the CNAME should point to.";
```

Comment:
Are the zones expected to have a trailing "."? Technically this is what an FQDN means but I think this should be called out if it is required since its a common misconception.

Reply:
I will test it with and without trailing `.` and if it's necessary I think I will add it on the backend side. But I have managed to create an A record with ipa cli without adding the `.` after the zone name or adding one before. (checked the sent request and the cli hasn't added the dot so I think we are good without it)